packages/mobile/src/components/AvatarSelf.tsx

```diff
@@ -36,5 +36,12 @@ export function AvatarSelf({ iconSize, displayNameStyle }: Props) {
     }
   }
 
-  return <Avatar recipient={recipient} iconSize={iconSize} displayNameStyle={displayNameStyle} />
+  return (
+    <Avatar
+      recipient={recipient}
+      iconSize={iconSize}
+      displayNameStyle={displayNameStyle}
+      address={account ?? undefined}
```

Comment:
Why is this necessary? IIUC either:
1. There will be recipient object with a valid `e164PhoneNumber` property, in which case `e164NumberToShow` in `Avatar.tsx` will be valid
2. There will be recipient object with a valid `address` property, so we can deconstruct and use it

Reply:
The `address` is used to use the right color for the avatar in case the user has no image.